app/lib/sortByLocalType.js

```diff
@@ -0,0 +1,35 @@
+const log = require('./logger');
+
+const localOrder = {
+  City: 5,
+  'Suburban Area': 4,
+  Town: 3,
+  Village: 2,
+  Hamlet: 1,
+  'Other Settlement': 0,
+};
+
+function getOrder(type) {
+  const order = localOrder[type];
+  if (order === undefined) {
+    log.error(`unknown local_type '${type}'`);
+    return -1;
+  }
+  return order;
+}
+
+function sortByLocalType(places) {
+  return places.sort((a, b) => {
```

Comment:
In order to improve the consistency of the order of places sorting a second time alphabetically would be helpful. What do you reckon?